### ISSUE TITLE
Y-axis static labels

### ISSUE THREAD

**Issue by bhoomika911** [indiespirit/react-native-chart-kit#156]
Y-axis labels are generating automatically. Can we give input for y-axis lables?

![image](https://user-images.githubusercontent.com/26411261/64487863-8ebc3000-d25d-11e9-855e-1e915b63dfa6.png)

**Comment by handoff** [comment]
Yes, it becomes very odd when showing something that can only be integers. Like 3.34534 individuals in my bar chart. :)

**Comment by yazmnh87** [comment]
> Yes, it becomes very odd when showing something that can only be integers. Like 3.34534 individuals in my bar chart. :)

So how do you add the labels?

**Comment by SimonPringleWallace** [comment]
Possible duplicate of #122?

**Comment by reflomdevelopment** [comment]
> Y-axis labels are generating automatically. Can we give input for y-axis lables?
> 
> ![image](https://user-images.githubusercontent.com/26411261/64487863-8ebc3000-d25d-11e9-855e-1e915b63dfa6.png)

how to make sticy y-axis on line chart
